submissions/chri911/popup-menu/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Popup</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <input type="checkbox" id="isexpanded">
+    <input type="checkbox" id="expanded">
+    <div class="container">
+        <div class="mail"><a href="#">Mail</a></div>
```

Comment:
Try to use more semantic HTML tags [Read more](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) Take a look at the navigation item.